### PR TITLE
[CH-190] enable tests in GlutenDataFrameAggregateSuite

### PR DESCRIPTION
**Pull request by binmahone**
Changelog category (leave one):
- New Feature
- Improvement
- Bug Fix (user-visible misbehaviour in official stable or prestable release)
- Performance Improvement
- Backward Incompatible Change
- Build/Testing/Packaging Improvement
- Documentation (changelog entry is not required)
- Not for changelog (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by kyligence-git** [comment]
Can one of the admins verify this patch?

**Comment by binmahone** [comment]
test this please with 545

**Comment by binmahone** [comment]
test this please with 545

File: utils/local-engine/AggregateFunctions/AggregateFunctionPartialMerge.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int ILLEGAL_TYPE_OF_ARGUMENT;
+    extern const int NUMBER_OF_ARGUMENTS_DOESNT_MATCH;
+}
+
+namespace
```
Comment (liuneng1994) [inline utils/local-engine/AggregateFunctions/AggregateFunctionPartialMerge.cpp:15]: 这个放到local_engine的namespace下面吧

File: utils/local-engine/AggregateFunctions/AggregateFunctionPartialMerge.h [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    extern const int ILLEGAL_TYPE_OF_ARGUMENT;
+}
+
+/**
+ * this class is copied from AggregateFunctionMerge with little enhancement.
+ * we use this PartialMerge for both spark PartialMerge and Final
+ */
+
+
+class AggregateFunctionPartialMerge final : public IAggregateFunctionHelper<AggregateFunctionPartialMerge>
```
Comment (liuneng1994) [inline utils/local-engine/AggregateFunctions/AggregateFunctionPartialMerge.h:25]: 同上，namespace

File: utils/local-engine/local_engine_jni.cpp [hunk 3]
```diff
@@ -270,7 +272,8 @@ jboolean Java_io_glutenproject_vectorized_BatchIterator_nativeHasNext(JNIEnv * e
 {
     LOCAL_ENGINE_JNI_METHOD_START
     local_engine::LocalExecutor * executor = reinterpret_cast<local_engine::LocalExecutor *>(executor_address);
-    return executor->hasNext();
+    bool x = executor->hasNext();
+    return x;
```
Comment (liuneng1994) [inline utils/local-engine/local_engine_jni.cpp:276]: 这里的作用是啥

**Review by liuneng1994 — CHANGES_REQUESTED** [review]
reviewed

File: utils/local-engine/local_engine_jni.cpp (hunk 3 above)
Comment (binmahone) [reply utils/local-engine/local_engine_jni.cpp:276]: 当时debug的时候改的 还原的时候没有100%还原 我来改一下

**Comment by binmahone** [comment]
test this please with 545

**Review by liuneng1994 — APPROVED** [review]
LGTM

**Pull request merged**